src/tv2-common/content/dve.ts

```diff
@@ -180,7 +183,7 @@ export function MakeContentDVE2<
 	dveGeneratorOptions: DVEOptions,
 	className?: string,
 	adlib?: boolean,
-	videoId?: string,
+	_videoId?: string,
```

Comment:
This seems weird to me. We have a parameter that we are not using??
If we are not using it we should delete it. If it turns out we need it later then we reimplent it. Untill then it's YAGNI